src/runtime/pkg/containerd-shim-v2/shim_management.go

```diff
@@ -307,8 +308,28 @@ func GetSandboxesStoragePath() string {
 	return "/run/vc/sbs"
 }
 
+// returns the storage path for runtime-rs created sandboxes
+// this is for consistency when using kata-runtime cli with runtime-rs
+func GetSandboxesStoragePathForRust() string {
```

Comment:
Why does runtime-rs need a different sandbox path? Can we just use the same sandbox path as the go runtime?

Reply:
@quanweiZhou PTAL

Reply:
Hi @bergwolf , I asked @quanweiZhou about this question.

The reason why runtime-rs changes `/run/vc/sbs` to `/run/kata` is that, in go runtime there are multiple directory under `/vc` for sandboxes. Using `/run/kata` will make it easier for resource cleanup.

But for compatability reason, using `/run/vc/sbs` will make it more transparent to the original 2.x developers.

IMO this needs some further discussion in the community.

Reply:
The go runtime has both `/run/vc/sbs` and `/run/vc/vm/` because there might not be any sandbox created when a VM is created (think of the vm caching and vm template features). Even if we switch `kata` to `vc`, we'd still need that extra layer to distinguish between sandboxes and vms. So I'd prefer that we keep using `/run/vc/` to avoid compatibility issue in `kata-runtime`.

Reply:
> The go runtime has both `/run/vc/sbs` and `/run/vc/vm/` because there might not be any sandbox created when a VM is created (think of the vm caching and vm template features). Even if we switch `kata` to `vc`, we'd still need that extra layer to distinguish between sandboxes and vms. So I'd prefer that we keep using `/run/vc/` to avoid compatibility issue in `kata-runtime`.

oh, that makes sense. I'll try to change that in runtime-rs and start testing:)

Reply:
I revert that commit, now it contains only the change of /agent-url in runtime-rs